### PR TITLE
fix Alexa.js issue and small ride-helper issue

### PR DESCRIPTION
**Pull request by JasonArkens17**

### CODE REVIEW

File: src/server/utils/ride-helper.js [hunk 1]
```diff
@@ -183,7 +183,9 @@ export const addRide = (ride, userId, origin, destination, cb) => {
   fetch(endpoint, {
       method: 'POST',
       headers: {
-        'Content-Type': 'application/json'
+        'Content-Type': 'application/json',
```
Comment (cpruijsen) [inline src/server/utils/ride-helper.js]: why is this duplicate?

File: src/server/utils/ride-helper.js [hunk 2]
```diff
@@ -153,7 +153,7 @@ export const addRide = (ride, userId, origin, destination, cb) => {
   let endpoint = process.env.PROD ? 'http://54.183.205.82/rides' : 'http://localhost:8080/rides';
 
   let body = {
-    userId: 3, // TODO: make this dynamic and not hardcoded once alexa auth is implemented
+    userId: 8, // TODO: make this dynamic and not hardcoded once alexa auth is implemented
```
Comment (cpruijsen) [inline src/server/utils/ride-helper.js]: why the change?

File: src/server/controllers/Ride.js [hunk 3]
```diff
@@ -118,7 +118,7 @@ export const helperAPIQuery = (body, vendor) => {
   if (vendor === 'Lyft') {
     let helperURL = CARVIS_HELPER_API + '/lyft/getCost';
   } else {
-    let helperURL = CARVIS_HELPER_API + '/uber/requestRide';
+    let helperURL = CARVIS_HELPER_API + '/uber/getEstimate';
```
Comment (daredia) [inline src/server/controllers/Ride.js]: how come this changed? we changed the name of the route? or are you pointing to a different route now than before?

Comment (JasonArkens17) [inline src/server/controllers/Ride.js]: Because now to order a ride with Uber it has to flow through getEstimate

Comment (JasonArkens17) [inline src/server/controllers/Ride.js]: It is reflected in the latest Uber flow that was submitted
 and merged yesterday

File: src/server/utils/ride-helper.js (hunk 2 above)
Comment (JasonArkens17) [inline src/server/utils/ride-helper.js]: This is for testing. Dont worry about it

File: src/server/utils/ride-helper.js (hunk 1 above)
Comment (JasonArkens17) [inline src/server/utils/ride-helper.js]: Its not a duplicate. I just copy pasted what you and Shez told me to from your previous PR....its the exact same thing? Whats the problem?

Comment (JasonArkens17) [inline src/server/utils/ride-helper.js]: Oh I see what you mean now. Fixing.

**Comment by daredia** [comment]
closing because changes are already addressed in #35
